apps/src/gamelab/GameLab.js

```diff
@@ -36,6 +41,13 @@ var GameLab = function () {
   this.tickIntervalId = 0;
   this.tickCount = 0;
 
+  /**
+   * Redux Store holding application state, transformable by actions.
+   * @private {Store}
+   * @see http://redux.js.org/docs/basics/Store.html
+   */
+  this.reduxStore_ = createStore(gamelabReducer);
```

Comment:
It's possible it would move upward at some point, but I'm not sure I'm convinced we're there yet.  I don't intend to backport redux into all of our classic puzzle types (I'm not sure the work is justified at this point) and applab/gamelab don't have a common ancestor that's not also shared with everything.

On the other hand, I feel like redux is lightweight and encapsulated enough that it might be better not to centralize it.  IMO one of our architectural problems is that we've been defining app types by inheritance rather than composition, which encourages duplication of any code that's needed by some but not all apps.  I'm trying to push us away from that with the top-level React work - apps should have unique top-level glue code, instantiating reusable components and tying them together.

The singular redux store feels (to me) like something that lives in that top glue-code layer - a page should only have one store, driven by one (composed) reducer that's probably unique to that app type.  The subreducers that make up the app-level reducer are probably reused though, which should minimize duplication in redux code too.